usr/src/lib/libfakekernel/common/cred.c

```diff
@@ -52,6 +53,34 @@ crgetuid(const cred_t *cr)
 	return (0);
 }
 
+/*ARGSUSED*/
```

Comment:
I'm not sure we should keep the cred stuff in libfakekernel.
Maybe better to leave that to the consumers?
(libfksmbsrv, and libzpool after this work)

Reply:
If we could add the missing functionality and make all consumers happy I'd rather do that.